### DISCUSSION TITLE
What's the best way to generate a headless screenshot from the SingleFile HTML?

### DISCUSSION THREAD

**Discussion by gzagatti** [gildas-lormeau/SingleFile#1317] [Q&A]
I would like to quickly generate a screenshot from the SingleFile HTML files to display as thumbnail in my terminal. 

For example to preview it from a CLI file manager like `nnn`. Is there a quick way to do that?

I have tried [`pageres`](https://github.com/sindresorhus/pageres-cli) but it's not super fast. Is there an alternative that you recommend?

**Comment by DejayRezme** [comment]
Now that I've started saving webpages with singlefile I'd also would like thumbnails for saved "zip.html" files to show up in windows explorer (or linux). But I'm not sure if this is supported by anything at all. Ideally small thumbnails could be saved when saving the singlefile.

PS: I really love this addon, it's brilliant!
